cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -967,8 +967,17 @@ def winnow_case_name(case_name: str) -> Set:
         "rel",
     }
 
-    # Remove all non alphanumeric characters
+    # Fix case name to be cleaner
     case_name = harmonize(case_name)
+
+    # Join abbreviations/acronyms. e.g. "D.L.M. v. T.J.S." -> "DLM v. TJS"
+    case_name = re.sub(
+        r"\b[a-zA-Z][a-zA-Z\.]*[A-Za-z]\b\.?",
+        lambda m: m.group().replace(".", ""),
+        case_name,
+    )
+
+    # Remove all non alphanumeric characters
```

Comment:
does this remove the chance that we will end up with case names that are blank?

Reply:
Should we add a check (log an error) if we reduce a title to nothing?

Reply:
A log could be useful to identify if that happens, I implemented it.